src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -75,10 +74,10 @@ export const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
     case VenueTemplate.animatemap:
       // NOTE: this is a must check for not spilling over global errors from animatemap onto other templates when it is unused
       template = isWebGl2Enabled() ? (
-        <Suspense fallback={LoadingPage}>
-          <AnimateMap space={venue} />
-        </Suspense>
+        // <Suspense fallback={LoadingPage}> //todo: fix this
+        <AnimateMap space={venue} />
       ) : (
+        // </Suspense>
```

Comment:
As the initial message, AnimateMap erros prevent use of all spaces, regardless if AnimateMap is used or not. To reproduce please check
- https://github.com/sparkletown/internal-sparkle-issues/issues/1523